code/modules/admin/verbs/event_triggers.dm

```diff
@@ -0,0 +1,121 @@
+/*
+Eventkit verb to be used to spawn the obj/effect/landmarks defined under code\game\objects\effects\landmarks_events.dm
+*/
+/client/proc/manage_event_triggers()
+	set name = "Manage Event Triggers"
+	set desc = "Open dialogue to create or delete narration/notification triggers"
+	set category = "EventKit"
+
+	if(!check_rights(R_FUN)) return
+
+
+
+	var/mode = tgui_input_list(src, "What do you wish to do?", "Manage Event Triggers", \
+	list(
+		"Create Notification Trigger",
+		"Create Narration Trigger",
+		"Manage Personal Triggers",
+		"Manage Other's Triggers",
+		"Cancel"
+	), "Cancel" )
+	if(mode == "Cancel") return
```

Comment:
Technicaly, nothing happens if you just cancel out of it, but still, having !mode check there is good practice I think.

Reply:
I'll make it if(!mode || mode == "Cancel") return then